update_workbook.py

```diff
@@ -27,21 +28,21 @@ class GitStatus(Enum):
 
 
 class Lecture(Enum):
+    PYTHONBASICS = auto()
+    PROPOSALS = auto()
     MCSTAS = auto()
+    SCIPP = auto()
     MODEL = auto()
-    PROPOSALS = auto()
-    PYTHONBASICS = auto()
     SCICAT = auto()
-    SCIPP = auto()
 
 
 lecture_to_directories = MappingProxyType({
-    Lecture.MCSTAS: Path('mcstas'),
-    Lecture.MODEL: Path('model'),
-    Lecture.PROPOSALS: Path('proposals'),
-    Lecture.PYTHONBASICS: Path('python'),
-    Lecture.SCICAT: Path('scicat'),
-    Lecture.SCIPP: Path('scipp'),
+    Lecture.PYTHONBASICS: Path('1-python'),
+    Lecture.PROPOSALS: Path('2-proposals'),
```

Comment:
this folder is now gone. Will the script break if the folder is not found or will it happily carry on?

Reply:
Then they will be ignored.
It's for filtering the directories that matter.